packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -52,11 +54,23 @@ export class OfficeAddinUsageData {
   private eventsSent: number = 0;
   private exceptionsSent: number = 0;
   private options: IUsageDataOptions;
+  private defaultData = {
+    Platform: process.platform,
+    NodeVersion: process.version
+  };
 
   constructor(usageDataOptions: IUsageDataOptions) {
     try {
-      this.options = usageDataOptions;
-
+      this.options = {
+        groupName: defaults.groupName,
+        promptQuestion: "",
+        raisePrompt: true,
+        usageDataLevel: UsageDataLevel.off,
+        method: UsageDataReportingMethod.applicationInsights,
+        isForTesting: false,
+        ...usageDataOptions
```

Comment:
I am confused - where does this input usageDataOptions get used?

Reply:
The properties of `usageDataOptions` gets written onto `this.options` on line 71.

Reply:
OK I just stepped through the code in VS - this makes sense to me now